src/main/java/src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,86 @@
+package src.main.java.duke;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+/**
+ * Represents the storage unit of the chat bot and contains all
+ * operations regarding the data storage of the bot
+ */
+
+public class Storage {
+
+    /**
+     * path of the file in the hard disk
+     */
+    private String path;
+    private TaskList list;
+
+    Storage(String directoryName, String fileName, TaskList list) {
+        this.list = list;
+        File directory = new File(directoryName);
+        if (!directory.exists()) {
+            try {
+                directory.mkdirs();
+            } catch (Exception error) {
+                System.out.println("Failed to create directory");
+            }
+        }
+        String path = directoryName + "/" + fileName;
+        File file = new File(path);
+        if (!file.exists()) {
+            try {
+                file.createNewFile();
+            } catch (Exception e) {
+                System.out.println("Failed to create a File");
+            }
+        }
+        this.path = directoryName + "/" + fileName;
+    }
+
+    /**
+     * method to read from the a text file on the system.
```

Comment:
Perhaps the header comment should start with a verb? e.g. Reads/Returns/Adds
I have attached the coding standard below
https://se-education.org/guides/conventions/java/intermediate.html
I noticed the same issue in multiple places